### DISCUSSION TITLE
Cloudflare Worker IP

### DISCUSSION THREAD

**Discussion by PikachuJP** [ImanMontajabi/CloudflareDNS#4] [General]
Step 1 : Create Cloudflare Account [Here](dash.cloudflare.com)
Step 2 : Create Worker From From Dashboard
Step 3 : Edit Source Code Of Worker With [This Code](https://getafreenode.com/blog/index.php/tutorial/32.html) Then Save & Deploy ( Copy The Worker URL We Need That Later ) ( Also You Can Add Your Domain From Worker - Trigger Panel )
Step 4 : Create New Worker From Dashboard ( It Can Be Anything Doesn't Matter What's The Name & Copy Worker URL We Need That Later Too )
Step 5 : Download V2Ray Android Client Or V2Ray Windows Client 
Step 6 : Add New Subscription From App Panel Use Your Link Like This ( https://XXX.XXX.workers.dev/sub/(Second Worker Address Or The Same First One Doesn't Matter Too ) 
For Example You Got Something Like This Link = https://firstip.testip.workers.dev/sub/secondip.testip.workers.dev
Or Something Like This = https://firstip.testip.workers.dev/sub/firstip.testip.workers.dev

This Post Will Update Ask Your Questions I Will Check Them Out !

**Comment by ImanMontajabi** [comment]
thanks

**Comment by PikachuJP** [comment]
Update #1 : On Desktop V2Ray Client You Have To Change The UTLS Fingerprint Option By Yourself It's Not Automatic Like Android App ( This Makes Huge Difference On Connection Stability And Speed )
